### PR TITLE
Add Your recent uploads in "more" menu

### PR DESCRIPTION
**Pull request by theefer**
Also replaces vertical dots with icon.

# Before

screenshot-2015-08-07t16 35 53

# After

screenshot-2015-08-07t16 37 36

### CODE REVIEW

File: kahuna/public/stylesheets/main.css [hunk 1]
```diff
@@ -1225,7 +1225,7 @@ FIXME: what to do with touch devices
     flex-basis: 45%;
 }
 
-.image-info__group--dl__value--full-metadata {    
+.image-info__group--dl__value--full-metadata {
```
Comment (jamesgorrie) [inline kahuna/public/stylesheets/main.css]: Always found it funny to call a class after an element.

**Comment by jamesgorrie** [comment]
:+1:

**Comment by alastairjardine** [comment]
I know it's merged, but could we make it a fully clickable button so that the target area is larger? IIRC the target area is quite a bit smaller than the other buttons? Create an issue? It'll be used a lot more now it's got some "real" functionality

**Comment by theefer** [comment]
Will take a look.

**Pull request merged**